integration-tests/tests.ts

```diff
@@ -1,30 +1,60 @@
-import {
-  test,
-  expect,
-  ConsoleMessage,
-  Page,
-  TestInfo,
-  BrowserContext,
-} from "@playwright/test";
+import { test, expect, ConsoleMessage, Page, TestInfo } from "@playwright/test";
 import fs from "fs";
 
-const BASE_URL = process.env.BASE_URL;
-const BWD_BASE_URL = process.env.BWD_BASE_URL;
-const options = {
-  baseURL: BASE_URL,
-  bwdBaseURL: BWD_BASE_URL,
-};
-test.use(options);
+import {
+  initMessages,
+  saveMessage,
+  getMessages,
+  clearMessages,
+} from "./messages";
+import {
+  canvasUrl,
+  awaitAnalysisLoaded,
+  awaitAnalysis,
+  bwdUrl,
+  caretPos,
+  createEmptyHTTPHandler,
+  createHTTPHandler,
+  createRepl,
+  createWorkerHandler,
+  expectContainsText,
+  expectExactText,
+  getElementSelectionStart,
+  getStyleProperty,
+  gotoAST,
+  gotoHash,
+  Locators,
+  get,
+  post,
+  pressShortcut,
+  selectAll,
+  waitForEmptyEntryBox,
+} from "./utils";
+
+export const BASE_URL = process.env.BASE_URL;
+export const BWD_BASE_URL = process.env.BWD_BASE_URL;
+
+declare global {
+  interface Window {
+    // makes TypeScript OK with us reaching into window.Dark.analysis
+    Dark: any;
+  }
+}
 
+/**
+ * Set local storage for userState and editorState. We don't want various UI
+ * elements to get in the way, including Fullstory consent, the "welcome to
+ * Dark" modal, etc.
+ */
 async function prepSettings(page: Page, testName: string) {
-  let setLocalStorage = async (key: string, value: any) => {
+  async function setLocalStorage(key: string, value: any) {
     await page.evaluate(
       ([k, v]) => {
         localStorage.setItem(k, v);
       },
       [key, JSON.stringify(value)],
     );
-  };
+  }
```

Comment:
I don't remember doing this - I think the formatter did it for me.

Reply:
Better to be consistent anyway.